File: Python/백준/Bronze/1259. 팰린드롬수/팰린드롬수.py
while True:
    n = input()
    if n=='0':
        break
        
    check = True
    for i in range(len(n)//2):
        if n[i] != n[len(n)-1-i]:
            check = False
            print("no")
            break
    if check:
        print("yes")

File: C++17/백준/Bronze/1259. 팰린드롬수/팰린드롬수.cc
#include<iostream>
#include<string>
using namespace std;

int main() {
	while (1) {
		string n;
		cin >> n;
		if (n[0] == '0')
			break;

		bool check = true;

		for (int i = 0; i < n.length()/2; i++) {
			if (n[i] != n[n.length() - i - 1]) {
				check = false;
					cout << "no" << endl;
					break;
				}
			}
		if (check)
			cout << "yes" << endl;
	}
}